src/functions/module_lookup.ts

```diff
@@ -813,7 +817,7 @@ export const PIVOT = {
     const pivotId = getPivotId(_pivotFormulaId, this.getters);
     const pivot = this.getters.getPivot(pivotId);
     const coreDefinition = this.getters.getPivotCoreDefinition(pivotId);
-    addPivotDependencies(this, coreDefinition);
+    addPivotDependencies(this, coreDefinition, coreDefinition.measures);
```

Comment:
So now we can reference headers of `PIVOT.HEADER()` but not headers originating from a spread pivot formula if I understand the code correctly. Do you think there's any way we can make that work ?

Reply:
> Do you think there's any way we can make that work ?

yes, but it'd be quite difficult I think. Every cell would need to be computed in the correct dependency order, when they are needed.